http/errors.go

```diff
@@ -12,12 +12,14 @@ package http
 
 import (
 	"encoding/json"
+	"fmt"
 
 	"github.com/sourcenetwork/defradb/errors"
 )
 
 const (
-	errFailedToLoadKeys string = "failed to load given keys"
+	errFailedToLoadKeys   string = "failed to load given keys"
+	errFailedToGetContext string = "failed to get context '%s'"
```

Comment:
suggestion: Similar to another comment I made on errors, please do not make error string dynamic like this. Move the context type to a KV instead.

Reply:
- [x] done